Valet/VALValet.m

```diff
@@ -450,6 +450,10 @@ - (BOOL)setObject:(nonnull NSData *)value forKey:(nonnull NSString *)key options
             keychainData[(__bridge id)kSecValueData] = value;
             
             status = VALAtomicSecItemAdd((__bridge CFDictionaryRef)keychainData, NULL);
+            
+            if (status == errSecMissingEntitlement) {
```

Comment:
I resisted using `CheckCondition` for two reasons: (1) The message it logs is prefixed with "Valet API misuse", which isn't actually true (and pretty misleading) in this case, and (2) I was trying to avoid making functional changes and introducing unnecessary risk.

I don't believe this is the same as #4 in the README, unless the README item is actually misdescribed and was this issue. I'm seeing this in Release as well as Debug, and once it starts happening it happens consistently until I run without the debugger attached.

I don't think adding this to `objectForKey:options:` would do anything since I think reading succeeds (it's just writing that fails). I can check again when I get back to my setup that reproduces this since I'm not positive on that.